publicHtml/htmlexpress.js

```diff
@@ -0,0 +1,13 @@
+const fs = require('fs');
+const express = require('express');
+const app = express();
+
+app.listen(3619);
+app.use(express.static('public'));
+
+app.get('/send', (req, res) => {
+  const name = req.query.Name;
+  const comment = req.query.Comment;
```

Comment:
change your variable names, make them different from your id, so that you wont get confused.